doc/tutorials/introduction_to_MNE.rst

```diff
@@ -0,0 +1,105 @@
+   - **Maxwell filtering** to remove environmental noise.
+   - **Boundary Element Modeling**: single and three-layer BEM model
+     creation and solution computation.
+   - **Forward modeling**: BEM computation and mesh creation
+     (see :ref:`ch_forward`)
+   - **Linear inverse solvers** (dSPM, sLORETA, MNE, LCMV, DICS)
+   - **Sparse inverse solvers** (L1/L2 mixed norm MxNE, Gamma Map,
+     Time-Frequency MxNE)
+   - **Connectivity estimation** in sensor and source space
+   - **Visualization of sensor and source space data**
+   - **Time-frequency** analysis with Morlet wavelets (induced power,
+     intertrial coherence, phase lock value) also in the source space
+   - **Spectrum estimation** using multi-taper method
+   - **Mixed Source Models** combining cortical and subcortical structures
+   - **Dipole Fitting**
+   - **Decoding** multivariate pattern analyis of M/EEG topographies
+   - **Compute contrasts** between conditions, between sensors, across
+     subjects etc.
+   - **Non-parametric statistics** in time, space and frequency
+     (including cluster-level)
+   - **Scripting** (batch and parallel computing)
+
+
+Is that all you can do with MNE-Python?
+---------------------------------------
+No!
+
+    - detect heart beat QRS component
+    - detect eye blinks and EOG artifacts
+    - compute SSP projections to remove ECG or EOG artifacts
+    - compute Independent Component Analysis (ICA) to remove artifacts or
+      select latent sources
+    - estimate noise covariance matrix from Raw and Epochs
+    - visualize cross-trial response dynamics using epochs images
+    - compute forward solutions
+    - estimate power in the source space
+    - estimate connectivity in sensor and source space
+    - morph stc from one brain to another for group studies
+    - compute mass univariate statistics base on custom contrasts
+    - visualize source estimates
+    - export raw, epochs, and evoked data to other python data analysis
+      libraries e.g. pandas
+    - and many more things ...
+
+
+What you're not supposed to do with MNE-Python
+----------------------------------------------
+
+    - **Brain and head surface segmentation** for use with BEM
+      models -- use Freesurfer.
+    - **Raw movement compensation** -- use Elekta Maxfilter™
+
+.. note:: This package is based on the FIF file format from Neuromag. It
+          can read and convert CTF, BTI/4D, KIT and various EEG formats to
+          FIF.
+
+
+Installation of the required materials
+---------------------------------------
+
+See :ref:`getting_started` with Python.
+
+.. note:: The expected location for the MNE-sample data is
+    my-path-to/mne-python/examples. If you downloaded data and an example asks
+    you whether to download it again, make sure
+    the data reside in the examples directory and you run the script from its
+    current directory.
+
+    From IPython e.g. say::
+    
+        cd examples/preprocessing
```

Comment:
see my recent changes to `getting_started`, use a `code-block` with `bash` language, and precede the line with `$` to make it clear it's a Bash command not Python